app/components/site-head.js

```diff
@@ -42,7 +40,10 @@ export default Component.extend(
           notTop: 'site-head-headroom-not-top',
         },
       };
-      this.setExperimentFlags();
+
+      // Hacky way to distinct UCP and App wikis as enableHydraFeatures is available only on UCP
+      const isUcpWiki = this.get('wikiVariables.enableHydraFeatures') !== undefined;
+      this.set('showSearchScope', isUcpWiki);
```

Comment:
Do we actually need to do that? Can we enable it for all wiki?

Reply:
Cross-wiki suggestions are not implemented on App ( UCP example https://ucp.fandom.com/wikia.php?controller=UnifiedSearchSuggestionsController&method=getSuggestions&format=json&scope=cross-wiki&query=ucp )